### PR TITLE
[BRAH-151] lyra pricing

### PR DESCRIPTION
**Pull request by Pradeep-selva**

### CODE REVIEW

File: contracts/LyraL2/LyraPositionHandlerL2.sol [hunk 1]
```diff
@@ -116,8 +133,9 @@ contract LyraPositionHandlerL2 is
         return
             (sUSDbalance * NORMALIZATION_FACTOR) /
             USDCPriceInsUSD +
-            IERC20(wantTokenL2).balanceOf(address(this)) +
-            address(this).balance;
+            (IERC20(wantTokenL2).balanceOf(address(this)) *
+                NORMALIZATION_FACTOR) /
+            1e6;
```
Comment (Bapi-Reddy) [inline contracts/LyraL2/LyraPositionHandlerL2.sol:138]: replace it with `USDC_NORMALIZATION_FACTOR`

File: hardhat.config.ts [hunk 2]
```diff
@@ -39,16 +39,15 @@ const buildForkConfig = (): HardhatNetworkForkingUserConfig => {
       blockNumber: 15003356,
     };
   }
-  console.log(forkMode);
   return forkMode;
 };
 
 const getTenderlyForkConfig = (): string => {
-  let url = JSON.parse(readFileSync("tenderlyConfig.json").toString());
+  // let url = JSON.parse(readFileSync("tenderlyConfig.json").toString());
```
Comment (Bapi-Reddy) [inline hardhat.config.ts:46]: fix tenderly fork generation later

Comment (Pradeep-selva) [reply hardhat.config.ts:46]: yeah we can scrap that and replace url later

File: contracts/LyraL2/LyraPositionHandlerL2.sol (hunk 1 above)
Comment (Pradeep-selva) [reply contracts/LyraL2/LyraPositionHandlerL2.sol:138]: ser we need the balance in 1e18 there thats why i did that

**Pull request merged**